### PR TITLE
Unified changelog

### PR DESCRIPTION
**Pull request by OskarStark**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,16 +6,16 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [3.1.0](https://github.com/sonata-project/SonataAdminBundle/compare/3.0.0...3.1.0) - 2016-05-17
 ### Added
-- Added AbstractAdmin, replacing Admin
+- Added `AbstractAdmin` class, replacing `Admin` class
```
Comment (Soullivaneuh) [inline CHANGELOG.md]: Too repetitive.

* Added `AbstractAdmin` class, replacing `Admin`?
* Added `AbstractAdmin` class, replacing `Admin` one?

Comment (OskarStark) [inline CHANGELOG.md]: > Added AbstractAdmin class, replacing Admin one? 

👍

Comment (OskarStark) [inline CHANGELOG.md]: done

**Comment by OskarStark** [comment]
hmm, looks like there is a problem with the commit signature.

here it is `Verified`, here not: https://github.com/sonata-project/SonataAdminBundle/commits/3.x

**Comment by OskarStark** [comment]
any idea @Soullivaneuh ?

**Comment by greg0ire** [comment]
The sha is not the same, because @Soullivaneuh did the actual commit with github, and there is currently no way to sign a commit from github UI

**Comment by OskarStark** [comment]
so we should merge instead of squash+commit imo

**Comment by Soullivaneuh** [comment]
> so we should merge instead of squash+commit imo

No. Instead, we should **use** squash+commit instead of doing merge except for PR having multiple **justified** commits.

This avoid to have the commit plus the ugly "Merge Pull Request #<number>" and keep a proper history. Just see: https://github.com/sonata-project/SonataAdminBundle/commits/3.x

BTW a note will be added on dev-kit later about it.

This is not our fault if GitHub does not sign automatically the commits. But feel free to open an issue about that: https://github.com/isaacs/github ;-)

**Comment by Soullivaneuh** [comment]
BTW: already done: https://github.com/isaacs/github/issues/643

**Pull request merged**